Templates/Per Tenant Extension/.github/workflows/TroubleShooting.yaml

```diff
@@ -0,0 +1,38 @@
+name: 'Troubleshooting'
+
+on:
+  workflow_dispatch:
+    inputs:
+      displayNameOfSecrets:
+        description: Display the name (not the value) of secrets available to the repository
```

Comment:
```suggestion
        description: Display the names (not the values) of secrets available to the repository
```

Reply:
Asked Susanne - she says it should be "Display the name (not the value) of secrets available to the repository" since there are multiple secrets, but they all just have one name and one value